x/permission/exported/proto.go

```diff
@@ -0,0 +1,16 @@
+package exported
+
+import (
+	"github.com/gogo/protobuf/proto"
+	"github.com/gogo/protobuf/protoc-gen-gogo/descriptor"
+)
+
+// GetPermissionRole returns the role that is defined for the given message. Returns ROLE_CHAIN_MANAGEMENT if none is set.
+func GetPermissionRole(message descriptor.Message) Role {
+	_, d := descriptor.ForMessage(message)
+	v, err := proto.GetExtension(d.GetOptions(), E_PermissionRole)
+	if err != nil {
+		return ROLE_CHAIN_MANAGEMENT
```

Comment:
I thought we want to give the strictest role by default, which is `ROLE_ACCESS_CONTROL` or in other words governance?

Reply:
When I asked I was told `ROLE_CHAIN_MANAGEMENT` was the strictest. So I'm fine either way, just need to know which 😆

Reply:
Oh, yeah, access control is the strictest one. Actually, this won't work, right? The AnteHandle is applied to Cosmos SDK/IBC messages too which don't have the descriptor set. So, this means that a user would need governance approval for a simple `MsgSend` 😆? Let's just leave it unrestricted by default and rely on our tests to catch unspecified roles for our messages perhaps?